INSTALL.md

```diff
@@ -104,7 +104,7 @@ For bare-metal installations, the full instructions for setting up PostgreSQL, T
     sudo apt-get install -y git
     git clone https://github.com/timescale/promscale_extension
     cd promscale_extension
-    git checkout 0.5.0
+    git checkout master
```

Comment:
Please don't change this. These instructions are intended as installation instructions for end users. We don't want end users installing off of `master`, that is just not a good idea.